### ISSUE TITLE
`chromium` mode does not actually use goma?

### ISSUE THREAD

**Issue by Dor1s** [google/clusterfuzz-tools#51]
I've launched the following command:
```
clusterfuzz reproduce 5083792420372480 -b chromium
```

and then noticed that my desktop became not too responsive during compilation. I see that the tool has started to build with **-j 480**:
```
$ ps aux | egrep ninja
mmoroz    93926 13.0  0.1  88380 76428 pts/34   R+   11:17   0:22 /usr/local/google/home/mmoroz/Projects/depot_tools/ninja-linux64 -w dupbuild=err -C /usr/local/google/home/mmoroz/Projects/new/chromium/src/out/clusterfuzz_builds -j 480 -l 480 v8_serialized_script_value_fuzzer
mmoroz   108396  0.0  0.0  22932   920 pts/94   S+   11:20   0:00 egrep --color=auto ninja
```

but GN args doesn't have `use_goma=true`:

```
$ cat /usr/local/google/home/mmoroz/Projects/new/chromium/src/out/clusterfuzz_builds/args.gn 
enable_nacl = false
ffmpeg_branding = "ChromeOS"
goma_dir = "/usr/local/google/home/mmoroz/goma"
is_debug = false
is_ubsan_security = true
optimize_for_fuzzing = true
pdf_enable_xfa = true
proprietary_codecs = true
use_libfuzzer = true
```

So it looks like the tool has kicked off build without goma. That explains why my little desktop with 24 cores became not responsive while running 480 build threads :)

**Comment by everestmz** [comment]
It looks like the `args.gn` for this testcase doesn't have `use_goma=true`, meaning the buildbots built without GOMA as well. We will take some time to decide whether it's more important to build faster (i.e. inject the goma parameters into the `args.gn`), or build exactly as the buildbot did.

**Comment by tanin47** [comment]
I think the right way here is to contact the owner of the build and ask her/him why goma isn't used.

If there's no reason against using goma, the owner should add it to `args.gn`.

This means we don't have to make any code change on our side (which is nice).

**Comment by Dor1s** [comment]
Can we use `-j$(nproc)` or some smaller number in cases like that? We cannot guarantee that `args.gn` always has `use_goma=true`, so we need to do either:
a) add that flag by ourselves
b) use another number of threads, e.g. `nproc`

**Comment by everestmz** [comment]
This kind of issue has been fixed by #66
